dags/glam_subdags/extract.py

```diff
@@ -135,18 +136,18 @@ def extract_user_counts(
     )
 
     gcs_delete = GoogleCloudStorageDeleteOperator(
-        task_id="glam_gcs_delete_count_extracts",
+        task_id="glam_gcs_delete_{}_extracts".format(task_prefix),
         bucket_name=glam_bucket,
-        prefix="glam-extract-firefox-counts",
+        prefix="glam-extract-firefox-{}".format(file_prefix),
         google_cloud_storage_conn_id=gcp_conn.gcp_conn_id,
         dag=dag,
     )
 
-    gcs_destination = "gs://{}/glam-extract-firefox-counts.csv".format(
-        glam_bucket
+    gcs_destination = "gs://{}/glam-extract-firefox-{}.csv".format(
+        glam_bucket, file_prefix
     )
     bq2gcs = BigQueryToCloudStorageOperator(
-        task_id="glam_extract_user_counts_to_csv",
+        task_id="glam_extract_user_{}_to_csv".format(task_prefix),
```

Comment:
I think you can remove `user` from here? The `task_prefix` already contains `user`

Reply:
Yes I will remove it. Thank you Marina!